### PR TITLE
2D polearm swing/thrust

### PR DESCRIPTION
**Pull request by Erdertus**
Second attempt at adding polearm 2D mode for swing over head and thrust only. 

Applied 2D moveset (swing overhead, stab thrust and no side swings) to Swiss Halberd and English Bill for live testing,

### CODE REVIEW

File: items/weapons.xml [hunk 1]
```diff
@@ -13140,31 +13140,31 @@
     </ItemComponent>
     <Flags />
   </Item>
-  <CraftedItem id="crpg_english_bill_v1_h0" name="{=Salt}English Bill" crafting_template="crpg_TwoHandedPolearm">
+  <CraftedItem id="crpg_english_bill_v1_h0" name="{=Salt}English Bill" crafting_template="crpg_Polearm_2dswing">
```
Comment (Yeldur) [inline items/weapons.xml:13143]: refund required

**Review by Yeldur — CHANGES_REQUESTED** [review]
refund required for each tier

**Review by Meowkov — APPROVED** [review]
This is looking fine by me and should in theory work.
Worst case we disable the items :D

**Review by Yeldur — APPROVED** [review]
Approved, falls in line with changes we want to make to enable 2D weapons

**Pull request merged**